raml/api.raml

```diff
@@ -0,0 +1,40 @@
+#%RAML 1.0
+title: Seecloud Security Service API
+version: v0.1
+mediaType: application/json
+types:
+  Subject:
+    type: object
+    additionalProperties: true
+    properties:
+      type: string
+      id: string
+    example:
+      id: "abc3978c-e0c9-4c56-8ed6-9d1d92bae110"
+      type: "securityGroup"
+      tenantId: "demo"
+      userId: "demo"
+  Issue:
+    type: object
+    properties:
+      issueType: string
+      description: string
+      discoveredAt: datetime
+      regionId: string
+      subject: Subject
+    example:
+      issueType: SecurityGroupTooOpen
+      description: "Security group too open"
+      discoveredAt: 2016-02-28T16:41:41.090Z
+      regionId: Region1
+      subject:
+        id: "d8b0be7c-2ad7-4083-8d5a-a7a9a56fdd14"
+        type: "securityGroup"
+        tenantId: "demo"
+        userId: "demo"
+/issues/{regionId}/:
```

Comment:
Part of API methods is missing: 

1) /regions -> returns list of regions names  (sample of implementation is here https://github.com/seecloud/health/blob/master/health/api/v1/regions.py) 

2) /issues -> then we don't filter by region (return from all regions) 

3) we need pagination (from-to) arguments 

4) we need reporting, it means we should have API that returns historical data about amount of new issues per region (or per all regions) (for last day, last week, last month). (just keep this in mind)